src/test/resources/com/adobe/epubcheck/bdd/general.feature

```diff
@@ -0,0 +1,7 @@
+Feature: General
+  As a user
+  I want the application to display information clearly
+
+  Scenario: [MUST] Visible version number
+    When I run the application with "--help"
```

Comment:
Should this be `--version` instead of `--help`? The test says "flag version."

Reply:
Hi @BigBlueHat

You're right. Today that functionallity is missing though. As mentioned in the pull request it can be changed when we decide how.to do with the --version.

Today the --version is undocumented feature to specify file version for some modes. We only documented -v so changing should not break anything.